process/coordinator/process.go

```diff
@@ -1034,8 +1034,8 @@ func (tc *transactionCoordinator) RequestMiniBlocks(header data.HeaderHandler) {
 
 	tc.requestedItemsHandler.Sweep()
 
-	crossMiniBlockHashes := header.GetMiniBlockHeadersWithDst(tc.shardCoordinator.SelfId())
-	for key, senderShardId := range crossMiniBlockHashes {
+	finalCrossMiniBlockHashes := tc.getFinalCrossMiniBlockHashes(header.GetMiniBlockHeadersWithDst(tc.shardCoordinator.SelfId()), header)
```

Comment:
you can remove first param and leave only header

Reply:
Done